.gitignore

```diff
@@ -18,3 +18,5 @@ docs/models
 docs/contribute.md
 docs/roadmap.md
 /.idea/
+
+installers/docker-compose/build
```

Comment:
Please update the doc links.
- Change docs/installation/installation-guide.md to point to installers
- Update installers/README.md to include an entry for docker-compose